app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,57 @@
+class RecipeFoodsController < ApplicationController
+  before_action :authenticate_user!, except: %i[index]
+  load_and_authorize_resource :recipe
+
+  def index
+    @recipe_foods = RecipeFood.include(:user).where(user_id: current_user.id)
+    # @all_foods = Food.includes(:user).where(user_id: current_user.id)
```

Comment:
- [ ] Please remove al unused code from your code so that your codebase remains neat and orgainsed.